mapsforge-map/src/main/java/org/mapsforge/map/layer/renderer/DirectRenderer.java

```diff
@@ -125,7 +126,8 @@ public TileBitmap executeJob(RendererJob rendererJob) {
         }
     }
 
-    private Set<MapElementContainer> processLabels(RenderContext renderContext) {
+    /** TODO (2024): If this renderer is ever going to be used, please see how labels are handled in the {@link DatabaseRenderer}, and then implement here. */
```

Comment:
`DirectRenderer` is not used by Mapsforge.

It could be used by apps to get bitmap tiles from vector maps in their own custom map engines.
However, it has worse map rendering than Mapsforge map engine, especially with the label layer.

It would be better if all developers work and improve the Mapsforge map engine,
so that our resources are not separated and we all benefit from the improvements.